Developer-Guide.md

```diff
@@ -176,6 +176,21 @@ option in the `plugins.linux` section of the `containerd` configuration file, su
     shim_debug = true
 ```
 
+### Enabling `crio` and `shiv2` debug
+
+In the `/etc/crio/crio.conf` file you'll be able to find:
```

Comment:
I'd mention also crio.conf.d/ which seems to override crio.conf if exist

Reply:
I've re-worded the content, please, let me know if the new content is fine to you.